Header header
nav_msgs/MapMetaData info
VerticalOccupancyGridList[] lists

